docs/reference/elasticsearch/index-settings/sorting.md

```diff
@@ -2,6 +2,9 @@
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-index-sorting.html
 navigation_title: Sorting
+applies_to:
+  stack: all
+  serverless: all
```

Comment:
I don't think you need the serverless tags on these settings if it's on the whole page, but I'm also ok with you keeping them.

Reply:
I left them to be consistent with the other pages, and because the banner says they will be marked, even if the entire page is marked as valid for serverless.

But I don't have a strong opinion either :S So, whatever you feel.

Reply:
I will say that on the similarity/analyzers page it isn't marked at the setting level so it's not 100% consistent. I know that those pages are a little different